dom/events/Event-initEvent.html

```diff
@@ -88,7 +88,6 @@
     assert_unreached("")
   }), false)
   assert_equals(target.dispatchEvent(e), true, "dispatchEvent must return true")
-  assert_equals(target.dispatchEvent(e), true, "dispatchEvent must return true")
```

Comment:
Hmm, no, this does need to be changed.  Per current spec, the first dispatchEvent() will unset the stopPropagation() flag, so it will hit the assert_unreached() in the event handler.  This should already be tested in Event-propagation.html, and has nothing to do with initEvent, so offhand I think it's fine to delete it.  The Gecko behavior that triggered a failure here also triggered a failure in Event-propagation.html, which is the proper file for this test.